README.md

```diff
@@ -144,14 +147,22 @@ export class Create {
   create () {
     this.requestInFlight = true;
```

Comment:
I know this is only your demo code - and it's not actually a change in this PR, but;
Is this flag actually ever checked against?